packages/web3-core-helpers/types/index.d.ts

```diff
@@ -69,6 +69,7 @@ export class errors {
     static InvalidResponse(result: Error): Error;
     static ConnectionTimeout(ms: string): Error;
     static ConnectionNotOpenError(): Error;
+    static ConnectionCloseError(event: WebSocketEvent | boolean): Error;
```

Comment:
Could you change the error handling to have the same signature as we have it with the other connection errors? 
``static ConnectionCloseError(event: WebSocketEvent | boolean): Error | ConnectionError;``

Reply:
I was wondering about that - this error is just a normal error with a message though - it doesn't have additional properties like `ConnectionError`.

Reply:
I've updated the ``errors.ConnectionCloseError`` type and the related function in the ``web3-core-helpers`` module.